core/src/main/java/org/apache/accumulo/core/security/crypto/FileEncrypter.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with this
+ * work for additional information regarding copyright ownership. The ASF
+ * licenses this file to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package org.apache.accumulo.core.security.crypto;
+
+import java.io.OutputStream;
+
+public interface FileEncrypter {
+  /**
+   * Encrypt the OutputStream.
+   *
+   * @since 2.0
```

Comment:
You can put `@since 2.0` at class level javadoc and all method will inherit it unless overridden.

Reply:
Ok cool.  Will do.